tests/accessor/generic_accessor_semantics.cpp

```diff
@@ -79,48 +79,100 @@ DISABLED_FOR_TEST_CASE(hipSYCL)
-        accessor_0, accessor_1,
-        "accessor<int, 1, access_mode::read_write, target::device>");
-  });
+  sycl_cts::util::get_cts_object::queue()
+      .submit([&](sycl::handler& cgh) {
+        auto accessor_0 = buffer_0.get_access<sycl::access_mode::read_write,
+                                              sycl::target::device>(cgh);
+        auto accessor_1 = buffer_1.get_access<sycl::access_mode::read_write,
+                                              sycl::target::device>(cgh);
+        common_reference_semantics::check_host<storage<1>>(
+            accessor_0, accessor_1,
+            "accessor<int, 1, access_mode::read_write, target::device>");
+      })
+      .wait_and_throw();
 }
 });
 
 DISABLED_FOR_TEST_CASE(hipSYCL)
 ("generic accessor common reference semantics, mutation (host)", "[accessor]")({
-  constexpr int new_val = 2;
+  sycl::queue queue = sycl_cts::util::get_cts_object::queue();
+  int result = 0;
+#ifdef SYCL_CTS_COMPILING_WITH_COMPUTECPP
   int val = 1;
+  int new_val = 2;
+#else
+  constexpr int val = 1;
+  constexpr int new_val = 2;
+#endif
   sycl::buffer<int> buffer{&val, sycl::range<1>{1}};
-  sycl::accessor<int, 1, sycl::access_mode::read_write, sycl::target::host_task>
-      t0{buffer};
 
   SECTION("mutation to copy") {
-    sycl::accessor<int, 1, sycl::access_mode::read_write,
-                   sycl::target::host_task>
-        t1(t0);
-    t1[0] = new_val;
-    CHECK(new_val == t0[0]);
+    {
+      sycl::buffer<int> buffer_result{&result, sycl::range<1>{1}};
+      queue
+          .submit([&](sycl::handler& cgh) {
+            auto acc_result = buffer_result.get_access(cgh);
+            sycl::accessor<int, 1, sycl::access_mode::read_write,
+                           sycl::target::host_task>
+                t0{buffer, cgh};
+            cgh.host_task([=] {
+              t0[0] = val;
+              sycl::accessor<int, 1, sycl::access_mode::read_write,
+                             sycl::target::host_task>
+                  t1(t0);
+              t1[0] = new_val;
+              acc_result[0] = t0[0];
+            });
+          })
+          .wait_and_throw();
+    }
+    CHECK(new_val == result);
   }
 
   SECTION("mutation to original") {
-    sycl::accessor<int, 1, sycl::access_mode::read_write,
-                   sycl::target::host_task>
-        t1(t0);
-    t0[0] = new_val;
-    CHECK(new_val == t1[0]);
+    {
+      sycl::buffer<int> buffer_result{&result, sycl::range<1>{1}};
+      queue
+          .submit([&](sycl::handler& cgh) {
+            auto acc_result = buffer_result.get_access(cgh);
+            sycl::accessor<int, 1, sycl::access_mode::read_write,
+                           sycl::target::host_task>
+                t0{buffer, cgh};
+            cgh.host_task([=] {
+              t0[0] = val;
+              sycl::accessor<int, 1, sycl::access_mode::read_write,
+                             sycl::target::host_task>
+                  t1(t0);
```

Comment:
```suggestion
                  t1{t0};
```
to be coherent with line 140?

Reply:
Done.